scripts/build-toolchains.sh

```diff
@@ -8,15 +10,83 @@ unamestr=$(uname)
 RDIR=$(pwd)
```

Comment:
nit: can we call `RDIR` something more descriptive like `INSTALL_DIR`

Reply:
Trying to keep it consistent with the firesim build-setup script?

Reply:
I personally like to error on verbosity rather than something like this... but if this doesn't change that is completely fine.

Reply:
I think in this case staying consistent is fine